app/views/guidance/guidance-for-vet.njk

```diff
@@ -0,0 +1,240 @@
+        </li>
+        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--dashed">
+          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#what-a-farmer-will-ask-you-to-do">Who can take part</a>
+        </li>
+        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--dashed">
+          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#arrange-a-review">Arrange a review</a>
+        </li>        
+        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--dashed">
+          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#during-the-visit">During the farm visit</a>
+        </li>
+        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--dashed">
+          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#after-the-review">After the review</a>
+        </li>                                
+    </ol>
+    
+    <div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6 govuk-!-margin-top-3">
+      <button onclick="window.print()" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
+    </div>
+
+    <section class="gem-c-devolved-nations">
+      <p>Before you visit a farm, print out this guidance on how to carry out a review, and the guidance on testing the species you’re reviewing:</p>
+       <ul class="govuk-list govuk-list--bullet">
+        <li><a href="/apply/test-cattle">cattle: testing required for an annual health and welfare review</a></li>
+        <li><a href="/apply/test-sheep">sheep: testing required for an annual health and welfare review</a></li>
+        <li><a href="/apply/test-pigs">pigs: testing required for an annual health and welfare review</a></li>
+      </ul>
+    </section>
+
+
+    <div class="govspeak govuk-!-margin-top-6">
+      <h2 id="what-the-review-is" class="govuk-heading-m">What the review is</h2>
+  
+      <p class="govuk-body">The annual health and welfare review of livestock is designed to help farmers:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>reduce endemic diseases and conditions</li>
+        <li>increase animal productivity</li>
+        <li>improve animal welfare</li>
+      </ul>
+
+      <p class="govuk-body">You may be asked by a farmer to carry out a review. It is not an inspection or audit and is optional.</p>
+
+      <h3 id="what-a-farmer-will-ask-you-to-do" class="govuk-heading-s">What a farmer will ask you to do</h3>
+
+      <ul class="govuk-list govuk-list--number">
+        <li>Visit the farm and test for the endemic diseases or condition of the livestock type you’re reviewing to recommended standards - bovine viral diarrhoea (BVD) in dairy and beef cattle, effectiveness of worming treatments in sheep, porcine reproductive and respiratory syndrome (PRRS) in pigs.</li>
```

Comment:
The style of the numbers within the list do not match the prototype which are white numbers in black circles

Reply:
Thanks - this one has been accepted as not possible with our component toolkit